crates/testlang/outputs/cargo/slang_testlang/src/generated/diagnostic.rs

```diff
@@ -0,0 +1,79 @@
+// This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+use std::borrow::Cow;
+
+use crate::text_index::TextRange;
+
+/// The severity of a diagnostic.
+///
+/// Explicitly compatible with the [LSP protocol](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#diagnosticSeverity).
```

Comment:
I wonder why `non_exhaustive` here, if we are "explicitly compatible with the LSP protocol".
My understanding is that the LSP's enum hasn't changed since at least 2017. Do you think they/we will ever add new variants here?
In both cases, adding a new one would be a "breaking change".

Reply:
Sure, I can drop this.

Reply:
Done in https://github.com/NomicFoundation/slang/pull/963/commits/3dba9deb7298471bef60f099acaf80d0c282f4d2